File: abc/abc165/abc165_b.fs
// https://atcoder.jp/contests/abc165/submissions/12651801

let X = stdin.ReadLine() |> int64

let rec solver m n =
    if m >= X then n
    else solver (int64 ((float m) * 1.01)) (n + 1)

solver 100L 0 |> stdout.WriteLine

File: abc/abc165/abc165_b.cs
// https://atcoder.jp/contests/abc165/submissions/12651452

using System;

class Program
{
    static void Main()
    {
        var X = long.Parse(Console.ReadLine());

        var m = 100L;
        var result = 0;
        while (m < X)
        {
            m += (long)(m * 0.01);
            result++;
        }
        Console.WriteLine(result);
    }
}
